src/deploy/osp_deployer/settings/config.py

```diff
@@ -480,20 +499,27 @@ def __init__(self, settings_file):
             'ovs_dpdk_enable']
         self.enable_ovs_dpdk = False
         if self.ovs_dpdk_enable.lower() == 'false':
-            pass
+            logger.info("OVS_DPDK is disabled.")
         elif self.ovs_dpdk_enable.lower() == 'true':
             self.enable_ovs_dpdk = True
-            for each in re.split(r'[_/]', self.nic_env_file):
-                if each.find('mode') != -1:
-                    self.ovs_dpdk_mode = each[-1:]
+            total_ports = 5
```

Comment:
is this used anywhere ?

Reply:
It was utilized earlier to compute the number of ports for OVS-DPDK, however, it hasn't been properly implemented yet. I'll remove this variable and the following commented out line.